### PR TITLE
Fix bug 1390111: Unescape \u0027 & \u0022 in Android

### PR DESCRIPTION
**Pull request by mathjazz**

### CODE REVIEW

File: pontoon/sync/tests/formats/test_silme.py [hunk 1]
```diff
@@ -221,8 +225,10 @@ def test_quotes(self):
         expected_string = dedent("""
             <!ENTITY SingleQuote "Single Quote \&apos;">
             <!ENTITY SingleQuoteEntity "Double Quote \&quot;">
+            <!ENTITY SingleQuotePython "\&apos;">
             <!ENTITY DoubleQuote '\&quot;'>
```
Comment (Pike) [inline pontoon/sync/tests/formats/test_silme.py]: The 'Double Quote "' should be inserted to [3], not [1] these days, right?

**Comment by mathjazz** [comment]
Thanks!

I addressed your comments and added a separate commit to fix bug 1390113 with a data migration.

**Comment by mathjazz** [comment]
@Pike Could you also review the fix for bug 1390113?

**Comment by mathjazz** [comment]
Thanks. I tested with the DB dump indeed. Everything worked as expected. Except that it was a little slow. Because `bulk_update` was running within the loop. :1st_place_medal:

Fixed and rebased.

**Pull request merged**